site/en/tutorials/images/transfer_learning.ipynb

```diff
@@ -337,7 +337,7 @@
         "id": "s9SlcbhrarOO"
       },
       "source": [
-        "Note: These layers are active only during training, when you call `model.fit`. They are inactive when the model is used in inference mode in `model.evaulate` or `model.fit`."
+        "Note: These layers are active only during training, when you call `model.fit`. They are inactive when the model is used in inference mode in `model.evaluate` or `model.fit`."
```

Comment:
Let's also make sure docs can be accessed by clicking on the methods (e.g. `Model.fit` -> https://www.tensorflow.org/api_docs/python/tf/keras/Model#fit):

```suggestion
        "Note: These layers are active only during training, when you call `Model.fit`. They are inactive when the model is used in inference mode in `Model.evaluate` or `Model.fit`."
```

This is similar to writing `tf.keras.Model.fit`, etc